### PR TITLE
ocs-to-ocs: correct logging

### PR DESCRIPTION
**Pull request by iamniting**
remove nigoyal from the logs which was left mistakenly during the
initial PR.

Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

**Comment by sp98** [comment]
Please add description to the PR and commit message.

File: controllers/storagecluster/provider_server.go [hunk 1]
```diff
@@ -155,12 +155,12 @@ func (o *ocsProviderServer) createSecret(r *StorageClusterReconciler, instance *
 	if err != nil && errors.IsNotFound(err) {
 		err = r.Client.Create(context.TODO(), desiredSecret)
 		if err != nil {
-			r.Log.Error(err, "nigoyal Failed to create Secret", "Name", desiredSecret.Name)
+			r.Log.Error(err, "Failed to create Secret", "Name", desiredSecret.Name)
```
Comment (sp98) [inline controllers/storagecluster/provider_server.go:158]:
```suggestion
			r.Log.Error(err, "Failed to create secret", "Name", desiredSecret.Name)
```

File: controllers/storagecluster/provider_server.go [hunk 2]
```diff
@@ -155,12 +155,12 @@ func (o *ocsProviderServer) createSecret(r *StorageClusterReconciler, instance *
...
 			return err
 		}
-		r.Log.Info("nigoyal Secret create succeeded", "Name", desiredSecret.Name)
+		r.Log.Info("Secret create succeeded", "Name", desiredSecret.Name)
 	} else if err != nil {
-		r.Log.Error(err, "nigoyal Failed to get Secret", "Name", desiredSecret.Name)
+		r.Log.Error(err, "Failed to get Secret", "Name", desiredSecret.Name)
```
Comment (sp98) [inline controllers/storagecluster/provider_server.go:163]:
```suggestion
		r.Log.Error(err, "Failed to get secret", "Name", desiredSecret.Name)
```

**Review by sp98 — CHANGES_REQUESTED** [review]
Nit

File: controllers/storagecluster/provider_server.go [hunk 3]
```diff
@@ -155,12 +155,12 @@ func (o *ocsProviderServer) createSecret(r *StorageClusterReconciler, instance *
 	if err != nil && errors.IsNotFound(err) {
 		err = r.Client.Create(context.TODO(), desiredSecret)
 		if err != nil {
-			r.Log.Error(err, "nigoyal Failed to create Secret", "Name", desiredSecret.Name)
+			r.Log.Error(err, "Failed to create Secret", "Name", desiredSecret.Name)
 			return err
 		}
-		r.Log.Info("nigoyal Secret create succeeded", "Name", desiredSecret.Name)
+		r.Log.Info("Secret create succeeded", "Name", desiredSecret.Name)
```
Comment (sp98) [inline controllers/storagecluster/provider_server.go:161]:
```suggestion
		r.Log.Info("Secret creation succeeded", "Name", desiredSecret.Name)
```

File: controllers/storagecluster/provider_server.go [hunk 4]
```diff
@@ -28,11 +28,11 @@ type ocsProviderServer struct{}
 func (o *ocsProviderServer) ensureCreated(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
 
 	if !instance.Spec.AllowRemoteStorageConsumers {
-		r.Log.Info("nigoyal Spec.AllowRemoteStorageConsumers is disabled")
+		r.Log.Info("Spec.AllowRemoteStorageConsumers is disabled")
 		return o.ensureDeleted(r, instance)
 	}
 
-	r.Log.Info("nigoyal Spec.AllowRemoteStorageConsumers is enabled creating resources")
+	r.Log.Info("Spec.AllowRemoteStorageConsumers is enabled creating resources")
```
Comment (sp98) [inline controllers/storagecluster/provider_server.go:35]:
```suggestion
	r.Log.Info("Spec.AllowRemoteStorageConsumers is enabled. Creating Provider API resources")
```

or specify which resources

**Review by sp98 — CHANGES_REQUESTED** [review]

File: controllers/storagecluster/provider_server.go (hunk 1 above)
Comment (iamniting) [reply controllers/storagecluster/provider_server.go]: done

File: controllers/storagecluster/provider_server.go (hunk 4 above)
Comment (iamniting) [reply controllers/storagecluster/provider_server.go]: done

File: controllers/storagecluster/provider_server.go [hunk 5]
```diff
@@ -28,11 +28,11 @@ type ocsProviderServer struct{}
 func (o *ocsProviderServer) ensureCreated(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
 
 	if !instance.Spec.AllowRemoteStorageConsumers {
-		r.Log.Info("nigoyal Spec.AllowRemoteStorageConsumers is disabled")
+		r.Log.Info("Spec.AllowRemoteStorageConsumers is disabled")
 		return o.ensureDeleted(r, instance)
 	}
 
-	r.Log.Info("nigoyal Spec.AllowRemoteStorageConsumers is enabled creating resources")
+	r.Log.Info("Spec.AllowRemoteStorageConsumers is enabled creating Creating Provider API resources")
```
Comment (sp98) [inline controllers/storagecluster/provider_server.go:35]:
```suggestion
	r.Log.Info("Spec.AllowRemoteStorageConsumers is enabled. Creating Provider API resources")
```

**Review by sp98 — CHANGES_REQUESTED** [review]
nit

File: controllers/storagecluster/provider_server.go (hunk 5 above)
Comment (iamniting) [reply controllers/storagecluster/provider_server.go]: done

**Review by sp98 — APPROVED** [review]

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**